.azure/pipelines/jobs/default-build.yml

```diff
@@ -91,7 +92,15 @@ jobs:
       ${{ if eq(parameters.agentOs, 'macOS') }}:
         vmImage: macOS-10.14
       ${{ if eq(parameters.agentOs, 'Linux') }}:
-        vmImage: ubuntu-16.04
+        ${{ if eq(parameters.useHostedUbuntu, true) }}:
+          vmImage: ubuntu-16.04
+        ${{ if eq(parameters.useHostedUbuntu, false) }}:
+          ${{ if eq(variables['System.TeamProject'], 'public') }}:
```

Comment:
oops 😈

Reply:
Hmm? Is that not correct?

Reply:
I meant this about your previous typo here.

Reply:
With enough force pushing there is no typo 😉